packages/at_chat_flutter/pubspec.yaml

```diff
@@ -2,9 +2,9 @@ name: at_chat_flutter
 description: A Flutter plugin project to provide a chat feature between atSigns built on the atPlatform to any Flutter application.
 version: 3.0.7
 homepage: https://docs.atsign.com/
-repository: https://github.com/atsign-foundation/at_widgets/tree/trunk/at_chat_flutter
+repository: https://github.com/atsign-foundation/at_widgets/tree/trunk/packages/at_chat_flutter
 issue_tracker: https://github.com/atsign-foundation/at_widgets/issues
-documentation: https://atsign.dev/docs/functional_architecture/libraries/#at_chat_flutter
+documentation: https://atsign.gitbook.io/docs/~
```

Comment:
We shouldn't use this documentation URL since it is a staging site. If we don't lose any points (which I don't think we will) we can remove the documentation URL